### PR TITLE
renamed byoei.* to connectors.es.*

### PR DESCRIPTION
**Pull request by tarekziade**
- `byoei.Fetcher` -> `connectors.es.sink.Extractor`
- `byoei.Bulker` -> `connectors.es.sink.Sink`
- `byoei.ElasticServer` -> `connectors.es.sink.SyncOrchestrator`

### CODE REVIEW

File: connectors/es/sink.py [hunk 1]
```diff
@@ -4,19 +4,17 @@
...
 #
 """
-Implementation of BYOEI protocol (+some ids collecting)
-
 `ElasticServer` is orchestrating a sync by:
 
 - creating a queue
-- launching a `Fetcher`, a wrapper on the top of the documents' generator
-- launching a `Bulker`, a class that aggregates documents and run the bulk API
+- launching a `Extractor`, a wrapper on the top of the documents' generator
```
Comment (timgrein) [inline connectors/es/sink.py:10]: From the doc string I would derive a name like `DocsGenerator`.

File: connectors/es/sink.py [hunk 2]
```diff
@@ -461,18 +459,18 @@ class ElasticServer(ESClient):
 
     - grabs all ids on Elasticsearch for the index
     - creates a MemQueue to hold documents to stream
-    - runs a `Fetcher` (producer) and a `Bulker` (consumer) against the queue
+    - runs a `Extractor` (producer) and a `Sink` (consumer) against the queue
```
Comment (timgrein) [inline connectors/es/sink.py:462]: We could also name both `DocsProducer` and `DocsConsumer`, but `Sink` is definitely better than `Bulker`, so also happy with that option.

File: connectors/es/sink.py (hunk 1 above)
Comment (seanstory) [reply connectors/es/sink.py:10]: "generator" is a python language term, but "DocsGenerator" would read to me as if we're fabricating the data (like Faker) as opposed to pulling the data from somewhere.

**Review by seanstory — APPROVED** [review]
"SyncOrchestrator" is 👨‍🍳 👌 

I still think that Extractor and SyncOrchestrator probably don't belong in `connectors.es.sync.py`, but that can be a followup.

**Pull request merged**